### DISCUSSION TITLE
Simultaneous fit of multiple 1D projections

### DISCUSSION THREAD

**Discussion by MarcelHoh** [scikit-hep/pyhf#2261] [General] [answered]
Dear experts,

In certain cases it can be useful to simultaneously fit several 1D projections of a data sample, to avoid complicated shapes in N dimensions or issues with low sample sizes. As far as I can tell this isn't supported in pyhf (or histfactory). Are there any plans to provide future support in pyhf? 
 
For an example see https://arxiv.org/pdf/1809.03290.pdf, which simultaneously fits 4 projections.
 
 Cheers,
 Marcel

**Comment by matthewfeickert** [comment]
From https://arxiv.org/abs/1809.03290:

> We use a set of one-dimensional projections of $w$, $\cos \theta_{\ell}$, $\cos \theta_{v}$, and $\chi$.

@MarcelHoh I didn't read the paper in full, but just jumping to that part and then looking at the plots of [Figure 5](https://ar5iv.labs.arxiv.org/html/1809.03290#S6.F5), e.g. 

[![Figure 5 w](https://ar5iv.labs.arxiv.org/html/1809.03290/assets/x9.png)](https://ar5iv.labs.arxiv.org/html/1809.03290#S6.F5)

, I would think that you would just implement these as each projection being its own HistFactory "[channel](https://pyhf.readthedocs.io/en/v0.7.2/intro.html#histfactory)" as you still have the [different processes](https://ar5iv.labs.arxiv.org/html/1809.03290/assets/x4.png) contributing as "samples". You of course need to think carefully about which modifiers are appropriate, but you need to do this regardless of what analysis you're doing.


> As far as I can tell this isn't supported in pyhf (or histfactory).

I'm responding quickly between meetings, so can you elaborate on this?

**Reply by alexander-held** [reply, marked as answer]
I assume the projections re-use the same data, so the model would need to model the statistical correlation (which the HistFactory model does not support).

**Reply by matthewfeickert** [reply]
> so the model would need to model the statistical correlation (which the HistFactory model does not support).

Ah yes, correct. My bad.

@MarcelHoh As @alexander-held pointed out that is indeed not supported and would need to be in some HistFactory `vX` for `X` $>1$ release in the future.

**Reply by MarcelHoh** [reply]
Thanks for the quick reply! Yes, the projections re-use the same data so the statistical correlations in the data and MC sample would need to be taken into account.
